packages/vite/src/node/server/hmr.ts

```diff
@@ -301,7 +301,13 @@ export const normalizeHotChannel = (
       }
       channel.on?.('vite:invoke', listenerForInvokeHandler)
     },
-    handleInvoke,
+    handleInvoke: async (payload) => {
+      const data = await handleInvoke(payload)
+      if (data.e) {
+        return { error: data.e }
+      }
+      return { result: data.r }
+    },
```

Comment:
Is there a reason we don't want `handleInvoke` to return complete `error` or `result` directly too? IIUC it'd break the `vite:invoke` event, but we're already breaking the naming convention in https://github.com/vitejs/vite/pull/18851

Reply:
It's to reduce the payload size. But I'm fine with making them consistent as I'm not sure about the perf difference.

Reply:
I think I'd lean on having consistent names as the payload savings probably isn't much compared to the data that the values can usually contain. But we can change it in a different PR if we'd like.